faqs.html

```diff
@@ -13,6 +13,18 @@ <h1 class="text-accent-4 center">Frequently Asked Questions</h1>
       <div class="collapsible-header black clemson-orange-text">When is the event?</div>
       <div class="collapsible-body"><span>CUCTF is taking place October 3rd, 2020.</span></div>
     </li>
+    <li>
+      <div class="collapsible-header black clemson-orange-text">COVID-19?</div>
+      <div class="collapsible-body">
+      <div class="card-content"> <span class="card-title clem"><i class="material-icons">error_outline</i></span>
```

Comment:
Inner `<div>` (and especially `card-content` since this isn't a card) does nothing here so just keep it all in the `collapsible-body` classed `<div>`.
```suggestion
      <span class="card-title clem"><i class="material-icons">error_outline</i></span>
```